### ISSUE TITLE
Keybind to toggle usage based CPU display

### ISSUE THREAD

**Issue by cooperuser** [ClementTsang/bottom#384]
## Describe the feature request

> From the README:
>
> ```text
>  -u, --current_usage                        Sets process CPU% to be based on current CPU%.
> ```

**Comment by ClementTsang** [comment]
I plan on building an in-app config screen that would let you toggle options currently in the flags/config file, which should hopefully cover your use case as well?

**Comment by cooper-anderson** [comment]
Yes that makes it easier, though having a quick keybind for it would be nice too, similar to the `%` mapping for toggling showing percents vs raw bytes

**Comment by ClementTsang** [comment]
Gotcha, I'll consider that as well.
